conan/tools/cmake/toolchain/blocks.py

```diff
@@ -215,6 +215,20 @@ def context(self):
         return {"arch_flag": arch_flag}
 
 
+class LinkerScriptsBlock(Block):
+    template = textwrap.dedent("""
+        string(APPEND CONAN_EXE_LINKER_FLAGS {{ linker_script_flags }})
+        """)
+
+    def context(self):
+        linker_scripts = self._conanfile.conf.get("tools.build:linker_scripts")
```

Comment:
```suggestion
        linker_scripts = self._conanfile.conf.get("tools.build:linker_scripts", check_type=list, default=[])
```
This way it validates it is always a list, to have consistent user inputs and better error messages.

Reply:
Thanks, done!